keras/backend/mxnet_backend.py

```diff
@@ -576,7 +580,7 @@ def eval(x):
     elif isinstance(x, KerasSymbol):
         executor = x.symbol.simple_bind(mx.cpu(), grad_req='null')
         for v in executor.arg_dict:
-            _bind_values[v].copyto(executor.arg_dict[v])
+            _bind_values[v].tensor.copyto(executor.arg_dict[v])
```

Comment:
maybe we can enum all symbols to see if there is tensor bind to it to avoid _bind_values.